python/packages/devui/pyproject.toml

```diff
@@ -4,9 +4,9 @@ description = "Debug UI for Microsoft Agent Framework with OpenAI-compatible API
 authors = [{ name = "Microsoft", email = "af-support@microsoft.com"}]
 readme = "README.md"
 requires-python = ">=3.10"
-version = "1.0.0b251001"
+version = "1.0.0-b251001"
```

Comment:
this doesn't have to be changed, and flit complains about the `-` so let's leave it out, once we do a release we'll update the version in one go